kokoro/config/build/presubmit/rockylinux8.gcl

```diff
@@ -3,7 +3,7 @@ import '../common.gcl' as common
 config build = common.build {
   params {
     environment {
-      // centos8 actually resolves to "rockylinux:8" in the Dockerfile:
+      // centos8 is built ontop of rockylinux:8 in Dockerfile:
```

Comment:
I liked this comment better before. Somewhere I think there's a TODO to rename centos8 (which is deprecated) to rockylinux8 everywhere, and this comment is part of that. It was saying that despite the fact that we pass in "centos8" as DISTRO, that doesn't actually mean we're building on CentOS8, we are actually building an RPM on rockylinux:8. So this "centos8" here is deceptive. Saying that "centos8 is built ontop of rockylinux:8" makes it sound like we are actually using centos8 in some way, which we are not.

Reply:
Reverting